presto-benchto-benchmarks/src/test/resources/sql/presto/tpcds/histogram/q85.plan.txt

```diff
@@ -0,0 +1,38 @@
+local exchange (GATHER, SINGLE, [])
```

Comment:
The only difference in calculations among all of the test cases in the code base were these two:

https://github.com/prestodb/presto/blob/838628b0574c3606302ba9e9f219b53c201d3c70/presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculatorHistograms.java#L31-L70

Due to the current design of the histogram API, the histograms don't use the same heuristics as the non-histogram calculation in cases with infinite bounds on one side. This only occurs for tables without statistics. It's mainly because the use of `Estimate` in the histogram API doesn't have a way of distinguishing `NaN` from an infinite bound, which creates ambiguity when determining which heuristic value to utilize.

Reply:
Also to confirm, when we disable using histogram, the behavior of current CBO will stay the same, is this correct?

Reply:
yes, the calculations stay the same as long as the flag is set to false (the default)